spec/models/showback_bucket_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'spec_helper'
+require 'money-rails/test_helpers'
+
+RSpec.describe ManageIQ::Consumption::ShowbackBucket, :type => :model do
+  let(:bucket) { FactoryGirl.build(:showback_bucket) }
+  let(:event) { FactoryGirl.build(:showback_event) }
+
+=begin
```

Comment:
Because there is a problem with the migration, when I changes _cents to _subunits, and Money does not recognize them. We will fix it when the migration is merged.